package.json

```diff
@@ -127,7 +127,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "tsc --noEmit --emitDeclarationOnly false && lint-staged",
```

Comment:
return to how this used to be, since the base tsconfig that is used by default has noEmit: true already set